test/jasmine/tests/mapbox_test.js

```diff
@@ -449,7 +449,7 @@ describe('mapbox plots', function() {
 
             var divStyle = mapInfo.div.style;
             ['left', 'top', 'width', 'height'].forEach(function(p, i) {
-                expect(parseFloat(divStyle[p])).toBeWithin(dims[i], 5);
+                expect(parseFloat(divStyle[p])).toBeWithin(dims[i], 8);
```

Comment:
`npm-run test-jasmine` runs without errors after this change (though that skips a bunch of the tests), but `citest-jasmine` still seems to fail.